IndexedDB/idbrequest_error.html

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+<meta charset=utf-8>
+<title>IDBRequest.error</title>
+<script src=/resources/testharness.js></script>
+<script src=/resources/testharnessreport.js></script>
+<script src=support.js></script>
+
+<script>
+async_test(t => {
+  var open = createdb(t);
+  open.onupgradeneeded = t.step_func(e => {
+    var db = e.target.result;
+    db.createObjectStore('store');
+  });
+  open.onsuccess = t.step_func(e => {
+    var db = e.target.result;
+    var request = db.transaction('store').objectStore('store').get(0);
+
+    assert_equals(request.readyState, 'pending');
+    assert_throws('InvalidStateError',
+                  function() { request.error; },
```

Comment:
Missed a `function` here? I think you might be able to collapse the line into the previous one if you use the arrow notation.